docker-compose-go.yaml

```diff
@@ -27,17 +27,35 @@ services:
       - hdised
 
   database:
-    build: ./postgres-config/
+    image: 'bitnami/postgresql:latest'
     container_name: database
-    volumes:
-      - ./postgresql.conf:/etc/postgresql/postgresql.conf
```

Comment:
We must build our custom image so we can use monitoring without the need to do a manual config